labels/V2.4/BirdNET_GLOBAL_6K_V2.4_Labels_ca.txt

```diff
@@ -1,6362 +1,6522 @@
-Cacicus haemorrhous_Cacic de carpó roig
-Cacicus latirostris_Cacic violaci
-Cacicus leucoramphus_Cacic montà septentrional
-Cacicus microrhynchus_Cacic de carpó escarlata
-Cacicus oseryi_Cacic d'elm
-Cacicus solitarius_Cacic solitari
-Cacicus uropygialis_Cacic subtropical
-Cacomantis castaneiventris_Cucut de ventre castany
-Cacomantis flabelliformis_Cucut cua de ventall
-Cacomantis leucolophus_Cucut de coroneta blanca
-Cacomantis merulinus_Cucut ploraner
-Cacomantis pallidus_Cucut pàl·lid
-Cacomantis passerinus_Cucut pitgrís
-Cacomantis sepulcralis_Rusty-breasted Cuckoo
-Cacomantis sonneratii_Cucut barrat
-Cacomantis variolosus_Cucut dels matolls
-Cairina moschata_Àenec mut
-Calamanthus fuliginosus_Espineta estriada
-Calamonastes simplex_Camaròptera grisa
+Buteogallus anthracinus_Aligot Negre Comú
+Buteogallus coronatus_Àguila Solitària Coronada
+Buteogallus meridionalis_Aligot de Sabana
+Buteogallus schistaceus_Aligot Amazònic
+Buteogallus solitarius_Àguila Solitària Negra
+Buteogallus urubitinga_Aligot Negre Gros
+Buthraupis montana_Tàngara Muntanyenca
+Butorides striata_Martinet Estriat
+Butorides virescens_notfound
+Bycanistes albotibialis_Calau Cuixablanc
+Bycanistes brevis_Calau Galtaargentat
+Bycanistes bucinator_Calau Trompeter
+Bycanistes fistulator_Calau Xiulador Occidental
+Bycanistes subcylindricus_Calau Galtagrís
+Cacatua alba_Cacatua Blanca
+Cacatua ducorpsii_Cacatua de les Salomó
+Cacatua galerita_Cacatua Crestagroga
+Cacatua goffiniana_Cacatua de les Tanimbar
+Cacatua sanguinea_Cacatua Sanguínia
+Cacatua tenuirostris_Cacatua Degollada
+Cacicus cela_Cacic de Carpó Groc
+Cacicus chrysonotus_Cacic Muntanyenc
+Cacicus chrysopterus_Cacic Alagroc
+Cacicus haemorrhous_Cacic de Carpó Roig
+Cacicus latirostris_Cacic Cuabarrat
+Cacicus oseryi_Cacic d'Elm
+Cacicus solitarius_Cacic Solitari
+Cacicus uropygialis_Cacic Subtropical
+Cacomantis castaneiventris_Cucut Ventre-roig
+Cacomantis flabelliformis_Cucut Cua de Ventall
+Cacomantis leucolophus_notfound
+Cacomantis merulinus_Cucut Ploraner
+Cacomantis pallidus_Cucut Pàl·lid
+Cacomantis passerinus_Cucut Pitgrís
+Cacomantis sonneratii_Cucut Barrat
+Cacomantis variolosus_Cucut de Matollar
+Cairina moschata_Ànec Mut
+Calamanthus fuliginosus_Espineta Estriada
+Calamonastes fasciolatus_Camaròptera Barrada
+Calamonastes simplex_Camaròptera Grisa
 Calamonastes stierlingi_Camaròptera de Stierling
-Calamospiza melanocorys_Sit blanc i negre
+Calamospiza melanocorys_Sit Blanc-i-negre
 Calandrella acutirostris_Terrerola de Hume
-Calandrella brachydactyla_Terrerola comuna
-Calandrella cinerea_Terrerola de casquet
+Calandrella brachydactyla_Terrerola Comuna
+Calandrella cinerea_Terrerola de Capell Rogenc
+Calandrella dukhunensis_Terrerola Oriental
 Calcarius lapponicus_Repicatalons de Lapònia
-Calcarius ornatus_Repicatalons pitnegre
+Calcarius ornatus_Repicatalons Pitnegre
 Calcarius pictus_Repicatalons de Smith
-Calendulauda africanoides_Alosa lleonada
+Calendulauda africanoides_notfound
```

Comment:
If there is no translation, please use the english name from the english label file

Reply:
@mrestivill updated these already in the latest commit